client/verta/docs/tutorials/organizations/create_organization.rst

```diff
@@ -0,0 +1,32 @@
+Creating an Organization
```

Comment:
After a brief discussion, let's just only capitalize the first letter of our section headers 😕 
```suggestion
Creating an organization
```